src/offset/local/mod.rs

```diff
@@ -95,33 +95,9 @@ impl Local {
     }
 
     /// Returns a `DateTime` which corresponds to the current date and time.
-    #[cfg(not(all(
-        target_arch = "wasm32",
-        feature = "wasmbind",
-        not(any(target_os = "emscripten", target_os = "wasi"))
-    )))]
-    #[must_use]
     pub fn now() -> DateTime<Local> {
         Utc::now().with_timezone(&Local)
     }
-
-    /// Returns a `DateTime` which corresponds to the current date and time.
-    #[cfg(all(
```

Comment:
The platform-specific code still lives in `Utc::now`. This could have been part of https://github.com/chronotope/chrono/pull/1072.
At the time of https://github.com/chronotope/chrono/pull/331 `Local::now` depended on `time::now`.